packages/turf-transform-rotate/test.js

```diff
@@ -45,8 +45,6 @@ test('rotate -- throws', t => {
     t.throws(() => rotate(featureCollection([line]), 100), /FeatureCollection is not supported/, 'featureCollection');
     t.throws(() => rotate(geometryCollection([line.geometry]), 100), /GeometryCollection is not supported/, 'geometryCollection');
```

Comment:
@DenisCarriere this currently fails because `geojson.type == 'Feature'` so it won't trigger the `Error`; was it working before?